### DISCUSSION TITLE
Sample code for execution

### DISCUSSION THREAD

**Discussion by nithinnani07** [saran-surya/cassata#1] [Q&A] [answered]
I have tried using this but not able to implement it. Any sample code please will be helpful.

@saran-surya

**Comment by saran-surya** [comment, marked as answer]
Hello @nithinnani07 , I have added a complete example for an express App, Kindly check them.

**Reply by nithinnani07** [reply]
bro, still its not working :(
![Uploading image.png…]()

**Reply by saran-surya** [reply]
Can you please tell what issue are you facing, I have tested it in HEROKU, United States based server, And I am able to proxy data.

**Reply by nithinnani07** [reply]
> Can you please tell what issue are you facing, I have tested it in HEROKU, United States based server, And I am able to proxy data.

Cannot GET /proxyouter

**Reply by nithinnani07** [reply]
```
const express = require('express');
const app = express();
const cassata = require("cassata");
cassata.proxySettings.roomId = "8754";
cassata.proxySettings.password = "8754";

// This will be your API end point that is only available in your location 
const API_END_POINT = "https://abc.com/" 

// Method 1 ~ Suitable for instant response.
app.get("/", async (req, res)=>{
    console.log("This is from your get request section");
    console.log(cassata.isProxyConnected());
    try {
        const response = await cassata.getProxiedData(API_END_POINT);
        // Process the response object here : {success: boolean, data : Response from API_END_POINT}
        res.send(response.success);
    } catch (error) {
        // Handle your errors here
    }
});

let proxyServer = cassata.createProxy(app);
proxyServer.listen(process.env.PORT || 3000, ()=>{
    console.log("Server started")
    // your initializations
});
```

**Reply by nithinnani07** [reply]
This is my code

**Reply by nithinnani07** [reply]
No, I don't have any other code only this mentioned above. could you please help with that router part.

**Reply by nithinnani07** [reply]
![image](https://user-images.githubusercontent.com/34380304/118258927-16941c00-b4ce-11eb-8e3d-09937eb8ad34.png)

**Reply by nithinnani07** [reply]
/proxyouter - I was using the same spelling from the readme docs :) 

Typo i guess

**Reply by saran-surya** [reply]
### Issue resolved.
Points to note : 
- Connect to the proxy client before accessing the data through CASSATA.getProxiedData()
- Provide a buffer time if you access them right from the start, (provides enough time for you to connect to the client)

### Changes made:
- Corrected spelling mistakes in the Readme.

Thanks @nithinnani07  for the discussion Happy coding :)

**Reply by nithinnani07** [reply]
Cool :)
